src/modm/platform/core/cortex/assert.cpp.in

```diff
@@ -84,7 +84,7 @@ void modm_abandon(const modm::AssertionInfo &info)
 {
 %% if target.platform in ["hosted"]
 	MODM_LOG_ERROR.printf("Assertion '%s'", info.name);
-	if (info.context != uintptr_t(-1)) { MODM_LOG_ERROR.printf(" @ %p (%lu)", (void *)info.context, info.context); }
+	if (info.context != uintptr_t(-1)) { MODM_LOG_ERROR.printf(" @ %p (%lu)", (void *)info.context, static_cast<long unsigned int>(info.context)); }
```

Comment:
```suggestion
	if (info.context != uintptr_t(-1)) { MODM_LOG_ERROR.printf(" @ %p (" PRIuPTR ")", (void *)info.context, info.context); }
```

This is I think the correct solution, it would work for aarch64 too.
http://www.cplusplus.com/reference/cinttypes/

Reply:
Fixed it